src/analog/message.ts

```diff
@@ -18,26 +18,21 @@ import getBase64FromBytes from '../utils/getBase64FromBytes.js';
 import mergeUint8Arrays from '../utils/mergeUint8Arrays.js';
 
 
-interface IMessageCommand {
-    /** command source binary data */
-    data: {
-        header: Uint8Array,
-        body: Uint8Array
-    },
-    /** specific command instance */
-    command: Command
+interface IMessageCommand extends ICommandBinary {
+    command: Command;
 }
 
-interface IMessage {
+export interface IMessage {
     commands: Array<IMessageCommand>,
     lrc: {
         expected: number | undefined,
         actual: number
     },
+    buffer: Uint8Array,
```

Comment:
```suggestion
    commandsData: Uint8Array,
```

?

it hints now that it shadows `commands` field

Reply:
fixed

Reply:
rename buffer to bytes